gateway-export-plugin/src/main/java/com/ca/apim/gateway/cagatewayexport/tasks/explode/linker/PolicyLinker.java

```diff
@@ -39,6 +39,16 @@
         return Policy.class;
     }
 
+    @Override
+    public void link(Bundle filteredBundle, Bundle bundle) {
+        Stream.of(
+                bundle.getEntities(Policy.class).values().stream(),
+                bundle.getEntities(GlobalPolicy.class).values().stream().peek(Policy.class::cast).collect(toList()).stream(),
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this use of "Stream.peek". ([squid:S3864](https://sonarcloud.io/organizations/ca-api-gateway/rules#rule_key=squid%3AS3864))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=com.ca.apim.gateway%3Agateway-developer-plugin&pullRequest=128&issues=AWcPsqoSd6oxh3R490-_&open=AWcPsqoSd6oxh3R490-_)

[ISSUE_KEY]: AWcPsqoSd6oxh3R490-_